apps/server/src/modules/migration/service/dto/page-content.dto.ts

```diff
@@ -0,0 +1,5 @@
+export class PageContentDto {
```

Comment:
why are the fields optional?

Reply:
to have partial responses. although that makes no sense anymore. changed